### ISSUE TITLE
App Status Sync Issue Across Multiple Apple TVs

### ISSUE THREAD

**Issue by pr0j3ctx** [jellyfin/Swiftfin#1155]
### Describe the bug

**Description:**

I am experiencing an issue with the Swiftin app on two Apple TVs. The app updates the watch status (watched/unwatched) only when I restart it. For instance, if I watch a movie in the living room and stop at 30 minutes, then switch to the bedroom and open the app that was running in the background, the movie does not appear under "Continue Watching". The status updates only after I restart the app.

**Steps to Reproduce:**

1.     Watch a movie on Apple TV 1 (e.g., in the living room).
2.     Stop the movie at a specific timestamp (e.g., 30 minutes).
3.     Switch to Apple TV 2 (e.g., in the bedroom) where the app is running in the background.
4.     Open the app from the background.

**Expected Behavior:**

The movie should appear under "Continue Watching" with the correct timestamp without needing to restart the app.

**Actual Behavior:**

The movie does not appear under "Continue Watching" until the app is restarted.

**Possible Solution:**

Implement a trigger to update the watch status when the app comes to the foreground from the background.

Please let me know if any further details are required.

### Application version

1.1

### Where did you install the app from?

App Store

### Device information

Apple TV 4K

### OS version

tvOS 17.5.1

### Jellyfin server version

10.9.7

**Comment by rwjack** [comment]
Similar bugs occur on a single ATV, there doesn't seem to be some sort of refresh function. E.g. when I add a new movie/show I have to close the app and re open-it in order for the new media to show. Hopefully we get the new release on ATV soon.

**Comment by Err0r0815** [comment]
I found another behaviour.
if i switch something to "watched" in the ATV App, the app will show the next episode after restarting the app :(.

**Comment by JPKribs** [comment]
Marking this as a duplicate of #784. This all stems from the same update issue so this is to centralize communication around a single issue and this issue was created first so I will be using that as the base.
